e2e/_suites/fleet/fleet.go

```diff
@@ -46,21 +47,18 @@ type FleetTestSuite struct {
 	FleetServerPolicy   kibana.Policy
 	Version             string // current elastic-agent version
 	kibanaClient        *kibana.Client
-
+	deployer            deploy.Deployment
```

Comment:
If this represents a deployment, let's name the variable as `deployment`. Or maybe we should rename the struct tp `Deployer` 🤔

Reply:
Let me work with the naming of things in a subsequent PR when I have more of the groundwork done for the installer portion